.github/workflows/wait-for-akd-publish.sh

```diff
@@ -0,0 +1,46 @@
+# Finds expected version of a crate in another crate's Cargo.toml file
+get_crate_expected_version_number()
+{
+    local crate_name=$1
+    local crate_to_look_for_version=$2
+
+    local cargo_toml_file="$crate_name/Cargo.toml"
+    local expected_version=$(grep "$crate_to_look_for_version = " $cargo_toml_file | tr -d '{^}' | awk -F '[,:=]' '{print $5}')
+    echo $expected_version
+}
+
+# Get published versions of a crate from https://github.com/rust-lang/crates.io-index/
+get_crate_published_versions()
+{
+    local crate_index_url=$1
+
+    local published_versions=$(
+        curl -sS "$crate_index_url" | jq .vers
+    )
+    echo "$published_versions"
+}
+
+# Wait for a specific akd version to be published to crates.io.
+# See https://github.com/novifinancial/akd/issues/116.
+# Must be run in the project root folder.
+akd_version_expected=$(get_crate_expected_version_number "akd_mysql" "akd")
+echo "AKD expected version in AKD_MYSQL:" $akd_version_expected
+wait_time=1
+while sleep $wait_time;
+do
+    akd_versions_published=$(
```

Comment:
If this _needs_ to be run from the repo root, you could ```pushd``` and ```popd``` it when you're done. We know the path of the script being run, so it should be do-able with relative paths. Something like
```bash
SCRIPTPATH=`dirname $SCRIPT` # Optionally you can do `dirname $0` since $0 is always the script path
pushd $SCRIPTPATH/../../
...do some work...
popd
```

example:
```bash
seanlawlor-mbp:SEEMless seanlawlor$ echo "$(dirname `pwd`)/../../"
/Users/seanlawlor/Code/../../
```

Reply:
CI already runs the script from the root repo. This is more of a comment on explicit use of the script, i.e., publishing a version that is not triggered with a release.